app/styles/nav-item.less

```diff
@@ -0,0 +1,33 @@
+.oss-navbar-item {
+  .border-radius-md;
+  padding: var(--spacing-px-6) var(--spacing-px-12);
+  display: flex;
+  flex-direction: row;
+  gap: var(--spacing-px-6);
+  align-items: center;
```

Comment:
maybe add the same transition as we have on the Sidebar::Item :
```
  transition: color 300ms ease-in-out, background-color 300ms ease-in-out;

```

Reply:
and maybe `cursor:pointer` as well ?

Reply:
Added transition : Split into i,span & the button class itself